libs/hpc-dummy/src/lib/dummy.ts

```diff
@@ -811,6 +811,12 @@ export class Dummy {
             };
           }
         ),
+        getFlowsDownloadXLSX: dummyEndpoint(
+          'flows.getFlowsDownloadXLSX',
+          async (params: flows.SearchFlowsParams) => {
```

Comment:
Remove `params` from this function if unused.

Reply:
Done